npm-shrinkwrap.json

```diff
@@ -1414,8 +1414,8 @@
     },
     "minimal-request": {
       "version": "2.1.1",
-      "from": "minimal-request@2.1.1",
-      "resolved": "https://registry.npmjs.org/minimal-request/-/minimal-request-2.1.1.tgz"
+      "from": "minimal-request@2.2.0",
+      "resolved": "https://registry.npmjs.org/minimal-request/-/minimal-request-2.2.0.tgz"
```

Comment:
I actually have done that on master already with #349 so I guess you just need to rebase from master here?